app/Console/Commands/WaitingLists/CheckMembersMeetActivityRules.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Events\Training\AccountMarkedForRemovalFromWaitingList;
+use App\Events\Training\AccountRegainedActivityRequirementsForWaitingList;
+use App\Models\NetworkData\Atc;
+use App\Models\Training\WaitingList;
+use Carbon\Carbon;
+
+class CheckMembersMeetActivityRules extends Command
+{
+    protected $minutesRequired = 720; // 12 hours is represented as 720 minutes
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:checkmembersmeetactivityrules';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Checks ATC waiting list activity rules are being met for each account';
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        WaitingList::all()->filter(function ($waitingList) {
```

Comment:
You're going to have an N+1 query problem here. You are accessing the accounts of a waiting list 'lazily,' which means you are doing more queries than necessary (see: https://laravel.com/docs/9.x/eloquent-relationships#eager-loading)

Reply:
Marking this as resolved now to clean up the PR, as this will be resolved by the scope change (https://github.com/VATSIM-UK/core/pull/2798#discussion_r1046254036) when the blocking PR is merged.